### ISSUE TITLE
kpt plot_predictions

### ISSUE THREAD

**Issue by HiiHongFe** [ultralytics/ultralytics#3226]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Validation

### Bug

When I'm validating the keypoint, it's throwing an error at the location of plot_predictions, as shown in the picture. I also tried your model and it also threw an error.
![image](https://github.com/ultralytics/ultralytics/assets/44940083/07f89152-7b83-4f3a-99f1-b76e47544cfb)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@HiiHongFe 👋 hi, thanks for letting us know about this possible problem with YOLO 🚀. 

Without more information and a reproducible example there's no action for us to take here.

We've created a few short guidelines below to help users provide what we need in order to start investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub with `git pull` or `pip install -U ultralytics` to ensure your problem has not already been solved in the latest code version.
- ✅ **Unmodified** – Your problem must be reproducible using official Ultrlaytics code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/ultralytics/issues/new/choose) with a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by glenn-jocher** [comment]
@HiiHongFe ah, I'm able to reproduce with this code. Adding a TODO to queue for a fix:

```python
from ultralytics import YOLO

model = YOLO('yolov8n-pose')
model.val(data='coco8-pose.yaml')
```

@Laughing-q @AyushExel looks like a Pose label plotting issue (doesn't affect training).

<img width="982" alt="Screenshot 2023-06-17 at 18 48 22" src="https://github.com/ultralytics/ultralytics/assets/26833433/8f674aad-56f3-446f-bd66-d5dbaaf23a6b">

**Comment by glenn-jocher** [comment]
Something seems to be incorrect about the input keypoints, they are the wrong shape:

```
cls.shape
(213,)
bboxes.shape
(213, 5)
masks.shape
(0,)
kpts.shape
(60, 17, 3)  < ----- PROBLEM
idx.shape
(213,)
```

**Comment by glenn-jocher** [comment]
@HiiHongFe Great news 😃! Your original issue may now be resolved ✅ in PR #3232. This will be part of our 8.0.119 release later this weekend.

To get this update:
- **[Git](https://github.com/ultralytics/ultralytics)** – Run `git pull` from within your `ultralytics/` directory or run `git clone https://github.com/ultralytics/ultralytics` again
- **[Pip](https://pypi.org/project/ultralytics/)** – Update with `pip install -U ultralytics`
- **[Notebooks](https://github.com/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb)** – Check out the updated notebooks  <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/ultralytics)** – Run `sudo docker pull ultralytics/ultralytics:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and letting us know. Please confirm if this update fixes the issue for you, and don't hesitate to report any other **issues** you find or **feature requests** you may have. Happy training with YOLOv8 🚀!
